### PR TITLE
Clarify payments eligibility

### LINKED ISSUE #112: Clarify guidelines even further
We did a lot of work in #45 to dial in the requirements for participation in the Pledge. We need to go further. Here's a list of cases on a spectrum surfaced (mostly; I added 7) by a potential Pledge member, with my take on what is and isn't acceptable to include when reporting Pledge payments:

1. **Yes** - Anonymous donation to OpenSSH - no recognition, no strings attached
2. **Yes** - Public donation to OpenSSH- very little benefit from recognition, no strings attached
3. **Yes** - Public donation to Drizzle ORM - a tool in our space (by donating publicly, we get some good will and recognition from prospects and customers), no strings attached
4. **No** - Funding a developer outside the company to work on an open-source library
5. **Probably** - Larger sponsorship of Drizzle - including the ability to request help from their team to work on a feature that makes the overall experience better for some of the company's customers.
6. **No** - Funding a developer outside the company to work on an open-source library that is integral to our company's success.
7. **No** - Funding a developer outside the company to work on an open-source project within our company's product ecosystem.
8. **No** - Funding development of specific open-source features at Drizzle - that benefit both the company and the broader open-source ecosystem.
9. **No** - Paying an invoice to Plausible - a commercialized non-vc-funded open source tool.
10. **No** - Paying an invoice to PostHog - a commercialized and vc-funded open source tool.

The "money value of time" line item on reports is intended to cover 4, 6, 7, and 8.

5 is a gray area, projects have sponsorship tiers, we recognize that. Also some projects are more important to a company than others (@schacon goes into this [in his post](https://blog.gitbutler.com/open-source-pledge-2024/#what-we-need)).

What does everyone think? We should reach some consensus on details like this and publish guidance on the website.

### PR DESCRIPTION
**Pull request by chadwhitacre**
Closes #112.

### CODE REVIEW

File: src/pages/about.astro [hunk 1]
```diff
@@ -103,12 +103,37 @@ const members = filterInactiveMembers(await getMembers());
...
+
+          <li>Payments to foundations are generally in scope, though most
+            foundations today are not set up to pay maintainers (they focus
+            more on conferences, trademarks, etc.). We hope that the Pledge
+            will enable foundations to shift towards paying maintainers in the
+            future.</li>
+
+          <li>Platforms such as thanks.dev, Open Collective, and GitHub
+            Sponsors are a common way to pay maintainers, though they can also
+            be used to make payments that are out-of-scope for the Pledge.</li>
```
Comment (vladh) [inline src/pages/about.astro:130]: Maybe it's worth clarifying that there are two separate things here. When using these platforms, there's the question of

1. _who_ is being paid — only recipients that fit the above criteria are eligible; and
2. _how much_ of the payment the recipient is receiving — minimal fees can be ignored, but the vast majority of money should go to maintainers.

For example, if I make a payment using thanks.dev, but tip thanks.dev 50% of the payment, I shouldn't be able to count the full payment.

File: src/pages/about.astro [hunk 2]
```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
...
         <h2>What payments are eligible?</h2>
 
-        <p
-          >Companies directly pay maintainers of Open Source projects they depend on, with no strings attached. Paying
-          developers to do specific work does not count towards the Pledge. Feel free to participate in <a
-          href="https://github.com/opensourcepledge/osspledge.com/issues/112">our ongoing discussion</a> on what counts
-          and what doesn't.</p
-        >
+        <p>The primary intent with the Pledge is <b>no-strings-attached payments
+        to community projects</b>. Start there.</p>
```
Comment (chadwhitacre) [inline src/pages/about.astro:107]: @ShaneCurcuru I changed "upstream" to "community" here.

File: src/pages/about.astro (hunk 1 above)
Comment (voxpelli) [reply src/pages/about.astro:134]: > if I make a payment using thanks.dev, but tip thanks.dev 50% of the payment, I shouldn't be able to count the full payment

A tricky part of this is that Open Collective (and GitHub Sponsors) are not as easy to quantify this for – Open Collective has one fee for the platform and then eg. the Open Source Collective has one fee for money flowing into the project

There's also often fees when withdrawing money, especially if one withdraws eg. Thanks.dev or GitHub Sponsors to the project's Open Collective.

Treating thanks.dev differently because it allows for a custom platform fee doesn't feel fair – especially as its the most transparent and simple approach and one I think should be encouraged rather than discouraged

File: src/pages/about.astro [hunk 3]
```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
...
+
+        <ol>
+
+          <li>Open Source projects that your company controls are out of scope.
+            If you own the trademark, it doesn't count.</li>
+
+          <li>Open Source projects that exclusively benefit your company or
+            your company's own ecosystem are out of scope.</li>
```
Comment (voxpelli) [inline src/pages/about.astro:115]: I find `exclusively benefit` here to be a bit hard to interpret – could it be clarified with an example?

File: src/pages/about.astro [hunk 4]
```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
...
+
+          <li>It's okay if you get non-exclusive logo placement or other
+            acknowledgement for your payments, or an hour or two of consulting
+            time with the project team, but anything beyond such minimal
+            benefits puts the payment out of scope.</li>
```
Comment (voxpelli) [inline src/pages/about.astro:120]: `an hour or two` should be in relation to the donation – the consulting time should be very minor in compared to the donation – a donation of `$100` to a project and a donation of `$10k` merits different minimal benefits

File: src/pages/about.astro (hunk 1 above)
Comment (chadwhitacre) [reply src/pages/about.astro:134]: Our goal for the Pledge over the next year should be to surface all of this sort of nuance, to bring clarity to the kinds of institutions in the ecosystem (companies, platforms, projects) and establish norms for transparency and accountability between all parties. That will be an ongoing conversation. Our goal right now is to get the conversation started.

File: src/pages/about.astro [hunk 5]
```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
...
+
+          <li>Payments to foundations are generally in scope, though most
+            foundations today are not set up to pay maintainers (they focus
+            more on conferences, trademarks, etc.). We hope that the Pledge
+            will enable foundations to shift towards paying maintainers in the
+            future.</li>
```
Comment (voxpelli) [inline src/pages/about.astro:126]: This is contradictory to `no-strings-attached payments to community projects` – I would reword this to explain why foundations are anyway acceptable

I would mention something about the non-profit / for-profit / non-commercial aspects of foundations @tieguy mentioned – else different people might interpret "foundation" differently, some thinking it refers to non-profits only, other thinking it includes wider support

File: src/pages/about.astro [hunk 6]
```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
...
+
+          <li>It's great if your company also actively participates in the
+            projects you fund, via code contributions, membership on an
+            advisory council or board, and the like.</li>
```
Comment (voxpelli) [inline src/pages/about.astro:130]: I'm missing a "but the time spent on that does not count towards the pledge for now"

File: src/pages/about.astro [hunk 7]
```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
...
+
+          <li>Platforms such as thanks.dev, Open Collective, and GitHub
+            Sponsors are a common way to pay maintainers, though they can also
+            be used to make payments that are out-of-scope for the Pledge.</li>
```
Comment (voxpelli) [inline src/pages/about.astro:134]: thanks.dev can not be used to make payments though?

I'm also missing a "so be aware that just because you're using one of these platforms that doesn't necessarily mean that your donation qualify for the pledge"

**Review by voxpelli — CHANGES_REQUESTED** [review]

File: src/pages/about.astro (hunk 3 above)
Comment (chadwhitacre) [reply src/pages/about.astro:115]: Fair enough, how about we drop it and go with this?

```suggestion
          <li>Open Source projects that exclusively benefit your company's
          own ecosystem are out of scope.</li>
```

**Comment by voxpelli** [comment]
I'm kind of having a hard time understanding whether the list is meant as an FAQ (it's not phrased as answers / questions) or a set of rules that a donation need to qualify for

File: src/pages/about.astro (hunk 7 above)
Comment (vladh) [reply src/pages/about.astro:134]: > thanks.dev can not be used to make payments though?

Sure it can! Maybe I'm misunderstanding?

File: src/pages/about.astro (hunk 4 above)
Comment (chadwhitacre) [reply src/pages/about.astro:120]: Better?

```suggestion
          <li>It's okay if you get non-exclusive logo placement or other
            acknowledgement for your payments, or an hour or two of consulting
            time along with a large sponsorship, but anything beyond such minimal
            benefits puts the payment out of scope.</li>
```

File: src/pages/about.astro (hunk 2 above)
Comment (chadwhitacre) [reply src/pages/about.astro]: Hrm ... changed again in bd7928b304412eff4cae823be3c3bf79f122b50c tho.

File: src/pages/about.astro (hunk 5 above)
Comment (chadwhitacre) [reply src/pages/about.astro]: > This is contradictory to "no-strings-attached payments to community projects"

Fair enough, resolved in bd7928b304412eff4cae823be3c3bf79f122b50c by shifting focus back to maintainers up above.

> different people might interpret "foundation" differently

Rather than clutter the language here further, I've brought back the link to [fossfoundation.info](https://fossfoundation.info/) in 
a0fc6359ecedee38a0a04c7f161605e91c9e9821. I'm with @ShaneCurcuru at https://github.com/opensourcepledge/opensourcepledge.com/issues/112#issuecomment-2395038843 that we should let the Pledge get established first before pushing too hard on the foundation question. I think it's enough that we briefly acknowledge just a bit of the complexity here.

File: src/pages/about.astro (hunk 7 above)
Comment (chadwhitacre) [reply src/pages/about.astro:135]:
```suggestion
          <li>Platforms such as thanks.dev, Open Collective, and GitHub
            Sponsors are a common way to pay maintainers, though some can also
            be used to make payments that are out-of-scope for the Pledge. Only
            payments that fit our criteria are in scope, regardless of payment
            platform.</li>
```

**Comment by chadwhitacre** [comment]
> I'm kind of having a hard time understanding whether the list is meant as an FAQ (it's not phrased as answers / questions) or a set of rules that a donation need to qualify for

Yeah I started by thinking this should be on /join but we already had a FAQ here, might make sense to move this to /join and link there from the FAQ.

File: src/pages/about.astro
Comment (tieguy) [inline src/pages/about.astro:1]: Gotta say I’m grumpy with this; the overwhelming majority of prominent open source foundations pay exactly 0% of their income to maintainers, and get this “generally in scope”, while Tidelift is getting dinged for… a hell of a lot more than 0%?

File: src/pages/about.astro (hunk 6 above)
Comment (chadwhitacre) [reply src/pages/about.astro:135]: Addressed in a separate point in d2c358e39fd198349031bf0405eeb8f07796805e.

File: src/pages/about.astro
Comment (vladh) [reply src/pages/about.astro:1]: @tieguy I totally see your point, and would love to continue discussing it, none of this is final. But I really do think not knowing what “a hell of a lot more than 0%” means does make it more difficult. Does that make sense?

Comment (chadwhitacre) [reply src/pages/about.astro:1]: Gotta say I'm grateful that you're here engaging with us about this. That's like 99% of what I see as the value of the Pledge, we have to have this conversation as an industry. I know OSS sustainability is the core mission of Tidelift and I would love to find a way to make it work for you all to be involved. I've avoided mentioning Tidelift in this PR precisely to keep our options open. 🙇‍♂️

What do you make of the transparency delta? Even a 501(c)(6) reports it's finances transparently, yes? So even if a trade association gives 0% today, we can all _see_ that, and negotiate / iterate towards a better balance. With Tidelift being a straight-up for-profit, we don't have transparency be default. Would Tidelift be willing to have a conversation about opening up its books to a parallel extent to say, LF?

Comment (chadwhitacre) [reply src/pages/about.astro:1]: I'll also point out that HeroDevs was [able to join the Pledge](https://github.com/opensourcepledge/opensourcepledge.com/pull/146) _even without_ their rev share being directly acknowledged. Could Tidelift do the same?

**Comment by chadwhitacre** [comment]
> Yeah I started by thinking this should be on /join but we already had a FAQ here, might make sense to move this to /join and link there from the FAQ.

Ftr I'll probably land this as-is as a FAQ update and then reshuffle /join in a future PR if I can get to it.

File: src/pages/about.astro
Comment (chadwhitacre) [reply src/pages/about.astro:1]: @tieguy I'm going to resolve this thread as Tidelift is not mentioned one way or another on this PR. I've cross-linked from https://github.com/opensourcepledge/opensourcepledge.com/issues/52, feel free to comment over there if/when you're ready to continue the conversation and we can reopen that issue. 🙏

File: src/pages/about.astro (hunk 4 above)
Comment (voxpelli) [reply src/pages/about.astro]: @chadwhitacre I would leave it out completely I think – if it isn't a minimal benefit / compensation then it doesn't matter if its one or ten hour – and depending on the donation one hour can be huge or ten hours can be small

If a company donates $10k monthly to a project, then eg. getting 10h per month of compensation is negligible

Also remember that without any compensatory benefits it can be hard to deduct the donation as a valid expense in some jurisdictions.

Comment (chadwhitacre) [reply src/pages/about.astro]: It's a common case, I think it's fine to mention as an example.

File: src/pages/about.astro (hunk 7 above)
Comment (voxpelli) [reply src/pages/about.astro]: @vladh How? Its for bulk sponsorship? Bulk sponsorship is hard to use as payment system? Especially as you can't set up any tiers (its like Flattr in that regard)

File: src/pages/about.astro (hunk 4 above)
Comment (voxpelli) [reply src/pages/about.astro]: @chadwhitacre Like worded now it sounds like a $50 donation can be compensated with the same consulting hours as a $10k though?

Any $50 donation that gives consulting hours should be considered a payment, as its a normal hourly rate.

So the example confuses more than it clarifies?

File: src/pages/about.astro (hunk 7 above)
Comment (vladh) [reply src/pages/about.astro]: Oh, maybe I'm misunderstanding what you mean by “payment system”. thanks.dev is a way to pay maintainers — multiple maintainers at once, on a schedule, to be sure, but you can pay maintainers using it. Did you mean instant one-to-one payments by “payment system”?

File: src/pages/about.astro (hunk 4 above)
Comment (chadwhitacre) [reply src/pages/about.astro]: Are we talking about the same text?

> or an hour or two of consulting time along with a large sponsorship

How does this make it sound like a $50 sponsorship can be compensated with the same consulting hours as a $10k sponsorship?

File: src/pages/about.astro (hunk 7 above)
Comment (voxpelli) [reply src/pages/about.astro]: Paying = strings attached
Donating = no strings attached

In Open Collective and GitHub Sponsors you can attach strings to the transactions, turning donations into payments. (And eg. polar.sh specializes in adding such strings)

File: src/pages/about.astro (hunk 4 above)
Comment (voxpelli) [reply src/pages/about.astro]: `large` is very hard to quantify – is it `$1k`, `$10k`, `$100k`? my point is that it should be relative to the donation, not fixed?

File: src/pages/about.astro (hunk 7 above)
Comment (chadwhitacre) [reply src/pages/about.astro]: > Paying = strings attached
> Donating = no strings attached

I don't like donating because it makes it sound like charity, which it's not. I would use "purchasing" for strings attached. We're inventing a new category so makes sense that language is unsettled yet. We should discuss elsewhere though: https://github.com/opensourcepledge/opensourcepledge.com/issues/173.

image

File: src/pages/about.astro (hunk 4 above)
Comment (chadwhitacre) [reply src/pages/about.astro]: Dude you're overthinking it, this illustrates the point that receiving minimal ancillary benefits doesn't discount payments that have the right intent.

File: src/pages/about.astro (hunk 7 above)
Comment (voxpelli) [reply src/pages/about.astro]: It was more in the context of this that I felt it was odd to mention thanks.dev:

> though some can also be used to make payments that are out-of-scope for the Pledge
